prds/done/177-project-setup-governance-tool.md

```
@@ -0,0 +1,1889 @@
+- File writing happens on client side using native tools
+- Git integration (commits, PRs) handled by client
+- Validation can be re-triggered by calling MCP with validation scope
+
+---
+
+**Removed**: ~~Stage 6: Integration~~ - MCP never writes files or creates commits directly
+
+### Template System
+
+**Directory Structure:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)